orchestra/workflow/load.py

```diff
@@ -0,0 +1,161 @@
+from django.db import transaction
+
+from orchestra.models import Certification
+from orchestra.models import Workflow
+from orchestra.models import WorkflowStep
+from orchestra.models import WorkflowVersion
+from orchestra.workflow.directory import parse_workflow_directory
+from orchestra.workflow.exceptions import WorkflowError
+
+
+@transaction.atomic
+def load_workflow(workflow_directory, versions=None, force=False):
```

Comment:
The `loadworkflow.py` script only passes in one, so I kind of agree. But the only complexity avoided by switching to a single version argument is the set logic in lines 59-69. Most of the complexity comes from the fact that when running the function (with any number of versions), the workflow might already be in the DB even if a specific version is not.